### ISSUE TITLE
Visible button (reminder) - Ships with no orders @Europe / Africa / Port Royale

### ISSUE THREAD

**Issue by Ignasito** [We-the-People-civ4col-mod/Mod#165]
Visible button (reminder) - Ships with no orders @Europe / Africa / Port Royale

**Comment by Nightinggale** [comment]
Another possible implementation would be to add a confirmation popup to end turn if there is a ship in Europe.

**Comment by Drunk-Monk** [comment]
Good idea. Another suggestion, is when a ship is at the edge of the map and you receive the prompt "Do you wish to travel to Europe?". The player could also receive the prompt travel to Africa / Port Royale.
